xbmc/platform/win10/WinRtUtil.h

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "HDRStatus.h"
+
+class CWinRtUtil
+{
+public:
+  CWinRtUtil(void);
```

Comment:
```suggestion
  CWinRtUtil();
```
and define on .cpp file or change here to `CWinRtUtil() = default;`

Same for  `~CWinRtUtil()` and `virtual` not seems necessary here

Reply:
I was lazy and c/p from WIN32Util.h but actually there is no need for a public constructor or destructor at this point. Could be added later when needed.